core/src/main/scala/cats/Foldable.scala

```diff
@@ -132,17 +137,44 @@ import simulacrum._
   def foldK[G[_], A](fga: F[G[A]])(implicit G: MonoidK[G]): G[A] =
     fold(fga)(G.algebra)
 
+
   /**
-   * find the first element matching the predicate, if one exists
+   * Find the first element matching the predicate, if one exists.
    */
   def find[A](fa: F[A])(f: A => Boolean): Option[A] =
-    foldLazy[A,Option[A]](fa, Lazy.eager(None)){ a =>
-      if(f(a))
-        Fold.Return(Some(a))
-      else
-        Fold.Pass
+    foldRight(fa, Lazy.eager(None: Option[A])) { a =>
+      if (f(a)) Fold.Return(Some(a)) else Fold.Pass
     }.value
 
+
+  /**
+   * Convert F[A] to a List[A].
+   */
+  def toList[A](fa: F[A]): List[A] =
+    foldLeft(fa, mutable.ListBuffer.empty[A]) { (buf, a) =>
+      buf += a
+    }.toList
+
+
+  /**
+   * Convert F[A] to a List[A], only including elements which match `p`.
+   */
+  def filter_[A](fa: F[A])(p: A => Boolean): List[A] =
+    foldLeft(fa, mutable.ListBuffer.empty[A]) { (buf, a) =>
+      if (p(a)) buf += a else buf
+    }.toList
+
+
+  /**
+   * Convert F[A] to a List[A], dropping all initial elements which
+   * match `p`.
+   */
+  def dropWhile_[A](fa: F[A])(p: A => Boolean): List[A] =
```

Comment:
Okay, that makes sense. If I have this question, I imagine others will too. I wonder if there's a good way to document it and make it easy to discover.